File: src/main/java/com/digsol/turnomed/dto/AddressDTO.java
package com.digsol.turnomed.dto;

import com.google.gson.JsonObject;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

import java.util.List;

@Data
@ApiModel(description = "Address")
public class AddressDTO implements BaseDTO {

    @ApiModelProperty(notes = "Address id")
    private Integer id;

    @ApiModelProperty(notes = "City name")
    private String city;

    @ApiModelProperty(notes = "Address")
    private String address;

    @ApiModelProperty(notes = "Clinic name")
    private String clinicName;

    @ApiModelProperty(notes = "Doctor types")
    private List<Integer> doctorTypes;

    @ApiModelProperty(notes = "Additional information")
    private JsonObject info;
}

File: src/main/kotlin/com/digsol/turnomed/dto/AddressDTO.kt
package com.digsol.turnomed.dto

import com.google.gson.JsonObject
import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty
import lombok.Data

@Data
@ApiModel(description = "Address")
data class AddressDTO(
        @ApiModelProperty(notes = "Address id")
        var id: Integer? = null,

        @ApiModelProperty(notes = "City name")
        var city: String? = null,

        @ApiModelProperty(notes = "Address")
        var address: String? = null,

        @ApiModelProperty(notes = "Clinic name")
        var clinicName: String? = null,

        @ApiModelProperty(notes = "Doctor types")
        var doctorTypes: List<Integer>? = null,

        @ApiModelProperty(notes = "Additional information")
        var info: JsonObject? = null
) : BaseDTO
